frontend/src/app/patients/personSchema.ts

```diff
@@ -123,7 +123,7 @@ export const personSchema: yup.SchemaOf<RequiredPersonFields> = yup.object({
   firstName: yup.string().required(),
   middleName: yup.string().nullable(),
   lastName: yup.string().required(),
-  birthDate: yup.string().required(),
+  birthDate: yup.date().required().max(new Date()).min(new Date("1/1/1900")),
```

Comment:
The validation text (specified lower in this file) doesn't make sense any more "Date of birth is missing or incorrectly formatted"

I recommend changing it to "Date of birth must be correctly formatted (MM/DD/YYY) and in the past"

Reply:
Yeah, make sense. But in that case, we do not inform that this is a mandatory field. only red asterisk near field.

Reply:
True! Maybe something like "Date of birth must be present, correctly formatted (MM/DD/YYY), and in the past"

Reply:
(Just seeing this now b/c of the incident report), but regarding the error copy, "Date of birth must be present," is redundant b/c an error implicitly states this, so I'd remove that part.

Note: we have a checklist item for design to review new content too, /cc @Rebecca-LM

Reply:
Even better would be an error message for when it's not present:
"Enter your date of birth"

And one when it's incorrectly formated: 
"Enter your date of birth in the correct format (MM/DD/YYYY)"

When they made both mistakes:
"Enter your date of birth in the correct format (MM/DD/YYYY) and in the past"

etc... you want the error to reflect whatever they did wrong so it's helpful.